app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -218,6 +218,9 @@ def render_unavailable
         message = I18n.t("curation_concerns.workflow.unauthorized")
         respond_to do |wants|
           wants.html do
+            if @presenter.blank?
```

Comment:
That said, it makes sense to have a really minimal presenter that only has the workflow state and the object type and label.